### PR TITLE
Feature/issue 9

### LINKED ISSUE #9: Port remaining scenarios over from haskell tester

### PR DESCRIPTION
**Pull request by bbangert**
Verified the forever looping actually works, fixed a bug for that.
Ported the repeating scenarios per issue 9.

Closes #9.

@jrconlin r?

### CODE REVIEW

File: aplt/runner.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def parse_statsd_args(args):
     client or None"""
     host = args.get("STATSD_HOST") or "localhost"
     port = int(args.get("STATSD_PORT") or 8125)
-    namespace = args.get("STATSD_NAMESPACE") or "aplt"
+    namespace = args.get("STATSD_NAMESPACE") or "push_test"
```
Comment (jrconlin) [inline aplt/runner.py]: +1 for more explicit name. (missed that last time, sorry)

File: aplt/runner.py [hunk 2]
```diff
@@ -283,7 +283,10 @@ def run_scenario(args=None, run=True):
     scenario = locate_function(arg)
     log.startLogging(sys.stdout)
     statsd_client = parse_statsd_args(arguments)
-    scenario_args = try_int_list_coerce(arguments["SCENARIO_ARGS"])
+    scenario_args = arguments["SCENARIO_ARGS"]
+    if scenario_args:
+        parts = [x.strip() for x in scenario_args.strip().split(",")]
+        scenario_args = try_int_list_coerce(parts)
```
Comment (jrconlin) [inline aplt/runner.py]: A bit confused here, why isn't this in try_int_list_coerce? You repeat the pattern for both instances of the call.

File: aplt/scenarios.py [hunk 3]
```diff
@@ -11,22 +11,85 @@
...
+                                                    60)
+        yield counter("notification.throughput.bytes", length)
+        yield counter("notification.sent", 1)
+        notif = yield expect_notification(reg["channelID"], 5)
+        yield counter("notification.sent", 1)
+        yield timer_end("update.latency")
+        yield ack(channel_id=notif["channelID"], version=notif["version"])
+        yield counter("notification.ack", 1)
+        yield wait(notif_delay)
+        if run_once:
```
Comment (jrconlin) [inline aplt/scenarios.py]: 46-57 gets repeated at least twice. Does it make sense to put it in a function if this is going to be a common pattern?

Comment (bbangert) [inline aplt/scenarios.py]: It'd be nice to be able to, which is why Python 3.3 added 'yield from' which lets you yield a generator itself. Unfortunately the semantics of doing that in Python 2.7 aren't quite as nice and it'd be messy.

**Comment by jrconlin** [comment]
nice fix. 

r+

**Pull request merged**